Userland/Libraries/LibELF/DynamicLinker.cpp

```diff
@@ -497,6 +514,10 @@ static void read_environment_variables()
         if (StringView { *env } == "_LOADER_BREAKPOINT=1"sv) {
             s_do_breakpoint_trap_before_entry = true;
         }
+
+        if (StringView { *env }.starts_with("LD_LIBRARY_PATH"sv)) {
+            s_ld_library_path = *env + 16u;   // Point the path to just the part after the '='
```

Comment:
I have a really bad feeling this is somewhat dangerous. Someone let me know if I've shit the bed on this one.

Reply:
environment variables are of the form of `VAR=VALUE` just by convention, that's not enforced anywhere, so this is not necessarily valid

Reply:
In that case, should we just strip as many characters until we reach a `/`? Feels like there's a _lot_ of permutations if this is the case.

Reply:
Just checking that the = is actually there should probably be enough :)